inc/languages/english/global.lang.php

```diff
@@ -414,6 +414,7 @@
 $l['error_boardclosed'] = "This bulletin board is currently closed. The Administrator has specified the reason as to why below.";
 $l['error_banned'] = "I'm sorry, but you are banned.  You may not post, read threads, or access the forum.  Please contact your forum administrator should you have any questions.";
 $l['error_cannot_upload_php_post'] = "Can not upload file - Too large for php post_max_size directive. Please press the back button.";
+$l['error_empty_post_input'] = "There has been an error due to your post data being empty. Possibly cause is a browser page refresh or direct access to this page. We recommend you press the browser back button and begin again.";
```

Comment:
Slight grammar tweak:

```
There has been an error due to your post data being empty. This could be due to a browser page refresh or direct access to this page. We recommend you press the browser back button and begin again.
```

Reply:
@euantorano - I am not a native English speaker so thank you for your reply :) should be fixed in my last commit